versioned_docs/version-1.0.0/advanced-topics/bufferserializer.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I can't seem to find where this document is accessible via the sidebar.
Are we referencing it somewhere else and/or did the sidebar not get updated?